### PR TITLE
Creates the site_metrics table via a database migration.

### PR DESCRIPTION
**Pull request by ashercodeorg**
Upcoming PRs will both add more columns to the table and establish a daily cron to insert rows into the table. For now, I'm staring with the skeleton of data that I've (mostly) been collecting manually.

### CODE REVIEW

**Comment by ashercodeorg** [comment]
@Hamms 

Note that this isn't urgent, submission can easily be postponed, if appropriate. Don't trust that I know what I'm doing (I don't) with respect to ActiveRecord migrations.

**Comment by Hamms** [comment]
It looks good to me, although I'm no ActiveRecord expert myself; @laurelfan might want to take a look.

You're creating the table without a primary key; can I assume you intend for this to be a one-row table? What is it going to be used by?

**Comment by ashercodeorg** [comment]
It'll be a very wide (lots of columns) table, with one row per day. So there will definitely be more than one row in the table. Perhaps the date column should be an (unique) index?

**Comment by Hamms** [comment]
A date index sounds great!

I'd also personally lean towards including a standard ID as well; even though there are often other ways to uniquely identify a row, using an ID means we don't have to think about this table as being fundamentally different from all the others.

File: dashboard/db/migrate/20151105100000_create_site_metrics.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+class CreateSiteMetrics < ActiveRecord::Migration
+  def change
+    create_table(:site_metrics, id: false) do |t|
+      t.date 'date'
```
Comment (philbogle) [inline dashboard/db/migrate/20151105100000_create_site_metrics.rb:4]: Does this table need an index on date?  It seems like we'll often want it scan date ranges.

**Comment by laurelfan** [comment]
I would probably include the id if you intend to interact with this table using activerecord at all. (and add a unique index on date as mentioned already)

**Comment by philbogle** [comment]
This PR can go in today once you're done with it; I don't see any scale risk until we start adding lots of data and querying it.

**Comment by philbogle** [comment]
Just reconfirming, LGTM

**Pull request closed without merging**